src/SpectatorView.Unity/Assets/SpectatorView.Editor/Scripts/CompositorWindow.cs

```diff
@@ -304,6 +309,182 @@ private void RecordingGUI()
+                        }
+                        EditorGUILayout.EndHorizontal();
+                        EditorGUILayout.BeginHorizontal();
+                        {
+                            GUIContent label = new GUIContent("S Offset");
+                            compositionManager.TextureManager.videoFeedColorCorrection.SOffset = EditorGUILayout.Slider(
+                                label,
+                                compositionManager.TextureManager.videoFeedColorCorrection.SOffset, -1, 1);
+                        }
+                        EditorGUILayout.EndHorizontal();
+                        EditorGUILayout.BeginHorizontal();
+                        {
+                            GUIContent label = new GUIContent("V Offset");
+                            compositionManager.TextureManager.videoFeedColorCorrection.VOffset = EditorGUILayout.Slider(
+                                label,
+                                compositionManager.TextureManager.videoFeedColorCorrection.VOffset, -1, 1);
+                        }
+                        EditorGUILayout.EndHorizontal();
+
+                        EditorGUILayout.Space();
+                        EditorGUILayout.BeginHorizontal();
+                        {
+                            GUIContent label = new GUIContent("Brightness");
+                            compositionManager.TextureManager.videoFeedColorCorrection.Brightness = EditorGUILayout.Slider(
+                                label,
+                                compositionManager.TextureManager.videoFeedColorCorrection.Brightness, -1, 1);
+                        }
+                        EditorGUILayout.EndHorizontal();
+                        EditorGUILayout.BeginHorizontal();
+                        {
+                            GUIContent label = new GUIContent("Contrast");
+                            compositionManager.TextureManager.videoFeedColorCorrection.Contrast = EditorGUILayout.Slider(
+                                label,
+                                compositionManager.TextureManager.videoFeedColorCorrection.Contrast, 0, 2);
+                        }
+                        EditorGUILayout.EndHorizontal();
+                        EditorGUILayout.BeginHorizontal();
+                        {
+                            GUIContent label = new GUIContent("Gamma");
+                            compositionManager.TextureManager.videoFeedColorCorrection.Gamma = EditorGUILayout.Slider(
+                                label,
+                                compositionManager.TextureManager.videoFeedColorCorrection.Gamma, 0.1f, 4);
+                        }
+                        EditorGUILayout.EndHorizontal();
+                    }
+                    EditorGUILayout.EndVertical();
+                }
+                else
+                {
+                    RenderTitle("Updating color correction is not possible when the compositor isn't running.", Color.green);
```

Comment:
>RenderTitle("Updating color correction is not possible when the compositor isn't running.", Color.green); [](start = 20, length = 105)

Same comment about not being able to change color correction without running.

Reply:
this one is already checked in, so i would say we could fix it in a follow up review.

Reply:
opened: https://github.com/microsoft/MixedReality-SpectatorView/issues/332